src/components/Login/ExistedAccountPwdSetting/index.tsx

```diff
@@ -0,0 +1,135 @@
+import ModalOperator from "@/components/Login/ModalOperater";
+import {
+  Button,
+  Divider,
+  Flex,
+  Icon,
+  ModalBody,
+  ModalHeader,
+  Text,
+  useToast,
+} from "@chakra-ui/react";
+import MainLogoSvg from "@/assets/svg/CourtCanva-main-LOGO.svg";
+import PwdInputGroup from "@/components/Login/PwdInputGroup";
+import React, { useState } from "react";
+import { updateUser } from "@/components/Login/helpers/userRequests";
+
+type Props = {
+  onClose: any;
+  setStep: React.Dispatch<React.SetStateAction<number>>;
+  userEmail: string;
+  currentStep: string;
+};
+
+const ExistedAccountPwdSetting: React.FC<Props> = ({
+  setStep,
+  onClose,
+  currentStep,
+  userEmail,
+}) => {
+  const handleCloseModal = () => {
+    setStep(1);
+    onClose();
+  };
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const [weakPasswordMsg, setWeakPasswordMsg] = useState("");
+  const toast = useToast();
+  const handleSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    if (password === "" || confirmPassword === "") {
+      setWeakPasswordMsg("");
+      setErrorMessage("Please fill all fields with asterisk!");
+      return;
+    }
+    if (password !== confirmPassword) {
+      setWeakPasswordMsg("");
+      setErrorMessage("Password does not match!");
+      return;
+    }
+    // regular expression from https://stackoverflow.com/questions/19605150/regex-for-password-must-contain-at-least-eight-characters-at-least-one-number-a
+    const passwordRegExp =
+      /^(?=.*\d)(?=.*[A-Z])(?=.*[a-z])(?=.*[ `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~])[a-zA-Z\d `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]{8,}$/; // eslint-disable-line
+    if (!passwordRegExp.test(password)) {
+      setErrorMessage("");
+      setWeakPasswordMsg(
+        "Password must contain at least 8 characters, one uppercase letter, one lowercase letter, one number and one special character!"
+      );
+      return;
+    }
```

Comment:
This part is the same as the _Login/Register.tsx_ Line 43-62, Should create a function in utils?